### PR TITLE
#584003: Upgrade to modelsdk latest.

### PR DESCRIPTION
**Pull request by ismailhabib**

### CODE REVIEW

File: mendixmodelcomponents.ts [hunk 1]
```diff
@@ -28,546 +28,545 @@ import {IModel, projects, domainmodels, pages, microflows, navigation, texts} fr
...
+	private _model: IModel;
 
-	mxModule.name = name;
+	constructor(model: IModel) {
+		this._model = model;
+	}
 
-	return mxModule;
-}
+	model(): IModel {
```
Comment (dlhartveld) [inline mendixmodelcomponents.ts]: Now that we're touching this kind of thing: Do you know/have you considered using a get/set property a la http://stackoverflow.com/questions/12827266/get-and-set-in-typescript ?

**Comment by ismailhabib** [comment]
@dlhartveld: CMIIW, I don't see any advantage of using a getter in this case.

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "typescript": "^1.6.2"
   },
   "dependencies": {
-    "mendixmodelsdk": "0.8.1",
+    "mendixmodelsdk": "../mendixmodelsdk/",
```
Comment (reinouts) [inline package.json]: 1.0.0

**Comment by reinouts** [comment]
In general I'm not too happy about requiring an instance of modelcomponents to be able to invoke methods on it. I've played around a little trying to create extension methods on (I)Model (cf. http://stackoverflow.com/questions/15478653/using-open-interfaces-to-extend-htmlelement-in-typescript) but due to time constraints I cannot pursue this further. 

Otherwise I think the changes are OK, just update the version number of mendixmodelsdk.

**Pull request merged**